### PR TITLE
[COR-1988] Update userAgent usage for android 10

### PR DESCRIPTION
**Pull request by hasretsariyer**

### CODE REVIEW

File: lib/webviewbridge.js [hunk 1]
```diff
@@ -440,7 +440,7 @@ function customNavigate(options) {
 
         var request = MutableRequest.requestWithURL(NSURL.URLWithString(targetUrl));
         cookie && request.addValueForHTTPHeaderField(`${cookie}`, "Cookie");
-        userAgent && webView.nativeObject.setValueForKey(userAgent, "customUserAgent");
```
Comment (ozcanovunc) [inline lib/webviewbridge.js]: This is a change for iOS, is it fine?

File: lib/webviewbridge.js [hunk 2]
```diff
@@ -453,9 +453,8 @@ function customNavigate(options) {
                 cookies[index] = c.trim();
                 c && cookieManager.setCookie(targetUrl, c);
             });
-
         }
-        userAgent && webView.nativeObject.getSettings().setUserAgentString(userAgent);
```
Comment (ozcanovunc) [inline lib/webviewbridge.js]: Is this change compatible with current live sf-core?

File: lib/webviewbridge.js (hunk 1 above)
Comment (hasretsariyer) [reply lib/webviewbridge.js]: yes, userAgent is a webview's property for both platforms

File: lib/webviewbridge.js (hunk 2 above)
Comment (hasretsariyer) [reply lib/webviewbridge.js]: compatible sf-core min version is 4.1.1. Coming soon :)

**Pull request merged**